conf/pulsar_env.sh

```diff
@@ -58,3 +58,16 @@ PULSAR_EXTRA_OPTS=${PULSAR_EXTRA_OPTS:-" -Dpulsar.allocator.exit_on_oom=true -Di
 
 #Wait time before forcefully kill the pulser server instance, if the stop is not successful
 #PULSAR_STOP_TIMEOUT=
+
+# Set BOOKIE options here to ensure that all pulsar scripts can work seamless with bookkeeper
+
+# default settings for starting bookkeeper
+
+# Memory size options
+BOOKIE_MEM=" -Xms2g -Xmx2g -XX:MaxDirectMemorySize=2g"
+
+# Garbage collection options
+BOOKIE_GC=" -XX:+UseG1GC -XX:MaxGCPauseMillis=10 -XX:+ParallelRefProcEnabled -XX:+UnlockExperimentalVMOptions -XX:+AggressiveOpts -XX:+DoEscapeAnalysis -XX:ParallelGCThreads=32 -XX:ConcGCThreads=32 -XX:G1NewSizePercent=50 -XX:+DisableExplicitGC -XX:-ResizePLAB"
+
+# Extra options to be passed to the jvm
+BOOKIE_EXTRA_OPTS="${BOOKIE_EXTRA_OPTS} ${BOOKIE_MEM} ${BOOKIE_GC} -Dio.netty.leakDetectionLevel=disabled -Dio.netty.recycler.maxCapacity.default=1000 -Dio.netty.recycler.linkCapacity=1024"
```

Comment:
maybe remove BOOKIE_MEM and BOOKIE_GC and just use PULSAR_MEM and PULSAR_GC? Because the only requirement is for people to add extra bookie opts (for kerberos), right?

Reply:
@sijie Yeah, I also found the jvm settings are the same and can be reused. Already changed, PTAL.